release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -116,6 +116,11 @@ With this release, there are several updates to the *Developer* perspective of t
 ** Cluster roles accessible to a project
 ** Actions visible on the *+Add* page
 ** Item types in the *Developer Catalog*
+* See the common updates to the *Pipeline details* and *PipelineRun details* page visualization performing the following actions:
+** Use the mouse wheel to change the zoom factor.
+** Hover over the tasks and see the task details.
+** Use the zoom in, zoom out, fit to screen, and reset view features using the standard icons.
+** *PipelineRun details* page only: At specific zoom factors, the background color of the tasks changes to indicate the error or warning status. Additionally, you can hover over the tasks badge to see the total number of tasks and tasks completed.
```

Comment:
Using "Additionally" is not required. It can be: You can hover over the tasks badge to see the total number of tasks and the completed tasks.

Reply:
+1 for Debargho's comments.